### DISCUSSION TITLE
Excluding values for HomeKit?

### DISCUSSION THREAD

**Discussion by ghubg10001** [itavero/homebridge-z2m#811] [Q&A] [answered]
I have the Aqara zigbee buttons and setup the Sonoff dongle on Zigbee2MQTT and then use this plugin to connect into HomeKit. It's awesome. However with Sonoff there are some exposed items I want to hide from HomeKit. One main one is each button has a temperature sensor, and I would like to hide that. Another is sometimes the little reset button shows as a programable option in HomeKit, so perhaps I want to hide that too. I am confused as to what values to put in where. 

An obvious starting point here would just be to focus on the temperature of the Aqara button and how I can exclude that so only the button is showing, I can figure out the rest from there. 

A follow-on question as well would be if you can change how things are categorized in HomeKit. With the temperature it shows under "climate" but hopefully removing that it can show under other of each room if that makes sense.

**Comment by itavero** [comment, marked as answer]
Please check the documentation for instructions on how to exclude certain properties: https://z2m.dev/config.html#devices

With regards to how things are "categorized" in the Home app: as far as I know there's not really anything a plugin can do about that.

**Reply by ghubg10001** [reply]
Thank you, I did follow those directions. I guess from looking at them I still can't figure out what the exlude_key is for temperature on that particular device. I did try to just exclude the entire device and that works, but I want the button part to show just not the temperature.

**Reply by ghubg10001** [reply]
Think I figured it out. First for the device ID I did the friendly name. Then for excluded values I did "device_temperature" which seemed to work. It's something in those settings.

**Reply by ghubg10001** [reply]
Also just one more note here, once I excluded "device_temperature" it then moved it from climate into other. All in all works exactly as I want it to now.
